### ISSUE TITLE
Remove cookie authentication

### ISSUE THREAD

**Issue by ekristen** [hashicorp/vault#563]
Anyone seen this?

```
2015/08/21 11:01:21 net/http: invalid byte '\n' in Cookie.Value; dropping invalid bytes
2015/08/21 11:01:22 net/http: invalid byte '\n' in Cookie.Value; dropping invalid bytes
```

**Comment by jefferai** [comment]
Thanks for the report -- cookies are going to be removed in 0.3 as they allow various forms of very bad behavior (a lot of it potentially accidental, which is even more worrying).

**Comment by jefferai** [comment]
Turning this into a milestone ticket to remind me to get that done.

**Comment by ekristen** [comment]
@jefferai is there a timeframe for 0.3.0 milestone?

**Comment by jefferai** [comment]
We are aiming for the second half of September, but I cannot provide a firm date.

**Comment by jefferai** [comment]
@ekristen This is now a deprecated in 0.3/remove in 0.4 target. But I'd suggest switching over as soon as you can.

**Comment by ekristen** [comment]
I will. Can we get tags on the repo? Would like to get homebrew updated with the newer 0.3 release. Thanks!

**Comment by jefferai** [comment]
@ekristen There already are tags on the repo for released versions. 0.3 isn't out yet.

**Comment by ekristen** [comment]
Is master considered 0.3?

**Comment by jefferai** [comment]
No, 0.3 is not done yet. Nothing will be 0.3 until it is officially tagged.

**Comment by jefferai** [comment]
OK, this is back to 0.3, see an upcoming post on the mailing list for details.

**Comment by ekristen** [comment]
+1

**Comment by jefferai** [comment]
Merged in #564.
